packages/html-reporter/src/metadataView.tsx

```diff
@@ -24,59 +24,61 @@ import './reportView.css';
 import './theme.css';
 
 export const MetadataView: React.FC<Metadata> = metadata => {
+  if (!metadata.ci?.link && !metadata.revision?.id && !metadata.revision?.author && !metadata.revision?.email && !metadata.revision?.subject && !metadata.revision?.timestamp && !metadata.revision?.link && !metadata.generatedAt)
```

Comment:
My gut tells me that we should make these flat with the named keys:
metadata['commit.author'] = ...

I think the list will grow beyond type-able once all kinds of Jira IDs get there...